_includes/post_footer.html

```diff
@@ -0,0 +1,3 @@
+<div class="post-footer">
+  <a href="/articles/">View all articles</a>
```

Comment:
should be `<a href="{{ site.baseurl }}/articles/">View all articles</a>` for consistency